types/prosemirror-model/index.d.ts

```diff
@@ -157,6 +157,13 @@ export class Fragment<S extends Schema = any> {
      * object with two separate positions is returned.
      */
     findDiffEnd(other: Fragment<S>): { a: number; b: number } | null | undefined;
+    /**
+     * : (number, ?number) → {index: number, offset: number}
+     * Find the index and inner offset corresponding to a given relative
+     * position in this fragment. The result object will be reused
+     * (overwritten) the next time the function is called. (Not public.)
```

Comment:
This method isn't public

Reply:
I got an error .

![image](https://user-images.githubusercontent.com/6762269/115512139-3875f680-a2b4-11eb-95e6-37225b846b42.png)

Reply:
but if the method is not public, that error is expected: you should not be using it

Reply:
Maybe  you should read it 。

https://discuss.prosemirror.net/t/prosemirror-code-with-typescript/3712/8

![image](https://user-images.githubusercontent.com/6762269/117755754-674a1180-b24f-11eb-96ac-a330aabfe7f8.png)